common/storybook/.storybook/decorators/withFeatureFlags.tsx

```diff
@@ -7,6 +7,18 @@ declare module '@storybook/types' {
     }
 }
 
+/** Sync with posthog/settings/feature_flags.py */
+const PERSISTED_FEATURE_FLAGS = [
```

Comment:
Why don't we just remove the feature flag since it's been rolled out to everyone?

Reply:
Will do in a follow-up PR, yup